support-frontend/app/controllers/RegularContributions.scala

```diff
@@ -88,12 +88,15 @@ class RegularContributions(
         InternalServerError
       },
       { statusResponse =>
-        if (!testUsers.isTestUser(request) && body.paymentFields.isDefined) {
-          monitoredRegion(body.billingAddress.country).map { region =>
-            val tipPath = TipPath(region, RecurringContribution, monitoredPaymentMethod(body.paymentFields.get), guestCheckout)
-            verify(tipPath, tipMonitoring.verify)
+        body.paymentFields.map(
```

Comment:
good improvement on my previous suggestion, although since we're side effecting foreach would be clearer
```suggestion
        body.paymentFields.foreach(
```